### ISSUE TITLE
Optimization of China area Kubernetes experience

### ISSUE THREAD

**Issue by hongxiaolu** [rancher/rancher#10323]
Hi, everybody
In rancher1.6.11, the old user experience is able to be improved by customizing the mirror repository and namespace, but new users still need to guide them to install properly. In order to solve this problem, we could synchronize the ADDONS's images from google official registry to Rancher's docker hub namespace. In the regard, we wrote a shell script to implement it.

Synchronous script : https://github.com/hongxiaolu/gcr_mirror_Script/tree/master

**Comment by hongxiaolu** [comment]
@nayang777

**Comment by hongxiaolu** [comment]
PR ： https://github.com/rancher/kubernetes-package/pull/116     https://github.com/rancher/rancher-catalog/pull/1009

**Comment by hongxiaolu** [comment]
New PR: https://github.com/rancher/rancher-catalog/pull/1021

**Comment by deniseschannon** [comment]
we've moved this out to the next release as switching from `gcr` to `dockerhub/rancher` as the default registry in the k8s template would cause add-ons to be re-created due to the image name change. We'll change it with the next major k8s release (k8s v1.9).

**Comment by yasker** [comment]
@deniseschannon Just a reminder for v1.9 release.

**Comment by hongxiaolu** [comment]
New PR  https://github.com/rancher/rancher-catalog/pull/1071

**Comment by hongxiaolu** [comment]
@nayang777  @deniseschannon

**Comment by deniseschannon** [comment]
@kingsd041 can you try to launch k8s `v1.9.2-rancher1-2` which is available in the v1.6-development branch of rancher behind the firewall? i've removed the references to GCR and would like that tested from china. it should only be pulling images from dockerhub now.

**Comment by kingsd041** [comment]
@deniseschannon 
Tested with Rancher/server:v1.6-development  with k8s v1.9.2-rancher1-2
I launched the agent in the cn-beijing region of Aliyun, and all the images needed to launch the agent were pulled from dockerhub.
```
# docker images
REPOSITORY                TAG                 IMAGE ID            CREATED             SIZE
rancher/k8s               v1.9.2-rancher1-2   019a0103ac5a        11 hours ago        1.669 GB
rancher/metadata          v0.10.2             77299bd2078e        8 days ago          245 MB
rancher/agent             v1.2.10-rc3         8133624860e7        9 days ago          237.2 MB
busybox                   latest              5b0d59026729        2 weeks ago         1.146 MB
rancher/network-manager   v0.7.20             eaf2cdb14863        3 weeks ago         255.8 MB
rancher/net               v0.13.9             8af43a9893d1        3 weeks ago         310.4 MB
rancher/dns               v0.17.2             349321911f08        3 weeks ago         243.4 MB
rancher/healthcheck       v0.3.3              14de771cc178        5 months ago        384.6 MB
rancher/etcd              v2.3.7-13           6c21cf015451        6 months ago        57.15 MB
rancher/net               holder              665d9f6e8cc1        10 months ago       267.2 MB
```
